Automattic/wp-calypso
# Log in: Fix woo login form divider style

Related to #84743

## Proposed Changes

* This https://github.com/Automattic/wp-calypso/pull/84551, but this one still happening in Woo log-in (see in screenshot the 'or' in the divider). This PR addresses it.

<img width="1320" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/5c361089-c8ba-4e6d-9b4f-e8eadce6c30c">


## Testing Instructions



* Pull and run this branch or use live link
* Navigate to `/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Dc92cfd0e05c2bf6e176a5ed1fbf185a99cf22a08cddf2bd631e1362b240d24f1%26redirect_uri%3Dhttps%253A%252F%252Fwoo.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252Fstart%25252F%252523%25252Finstallation%2526source%253Dsignup_menu%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=50916`
* Make sure it looks good
* Please also check if no regressions were added by checking https://github.com/Automattic/wp-calypso/pull/84551

## Pre-merge Checklist
